packages/elements/code-block/src/types.ts

```diff
@@ -1,6 +1,10 @@
 import { InsertNodesOptions } from '@udecode/plate-common';
 import { PlatePluginOptions } from '@udecode/plate-core';
 
+export interface CodeBlockNodeData {
+  lang: string;
```

Comment:
It may be better to make it optional, where an undefined value would be the default language (e.g. plain text). Also for backward compatibility